syntax = "proto3";

service FileService {
  rpc UploadFile(FileRequest) returns (FileResponse) {}
}

message FileRequest {
  string filename = 1;
  bytes content = 2;
  string file_type = 3;
}

message FileResponse {
  int64 size = 1;
  bool is_valid = 2;
  string message = 3;
}